core_protocol_working_group/meetings/meeting_template.md

```diff
@@ -0,0 +1,23 @@
+
+# Meeting 1
+
+## Date: Feb 14th, 2024, Wednesday, 9-10 AM PST
+
+## How to add an agenda topic?
+Any attendee can add an agenda topic by creating a pull request on this file and adding more topics in the agenda below.
+
+## Agenda
+
+1. OKRs for this cycle
```

Comment:
Just wanting to confirm that this is flexible. There are OKRs which we don't want to share publicly and there are OKRs, where the WG involvement has limited benefits. 

We have listed the Roadmap in our landing page: https://github.com/onflow/Flow-Working-Groups/tree/main/core_protocol_working_group  I am happy to talk through them. But for the working groups, I think we should list the areas that make sense for the working group to focus on. They'll be related to our OKRs but can be broader or much more specific than the OKR.

Reply:
yes - the plan if to share during the call the most relevant OKRs at a higher level.